src/actions/AccountReferralActions.ts

```diff
@@ -243,10 +249,9 @@ export const getCountryCodeByIp = async (): Promise<string | undefined> => {
  */
 async function saveAccountReferral(state: RootState): Promise<void> {
   const { account } = state.core
-  const { accountReferral, activePromotions } = state.account
+  const { accountReferral } = state.account
   const referralData = {
-    ...accountReferral,
-    activePromotions
+    ...accountReferral
```

Comment:
You can skip this copying step now. Just put this function back to the way it is in `develop`.